app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.test.tsx

```diff
@@ -146,31 +146,6 @@ describe("FormBranch", () => {
     await assertInvalidField(container, "referenceBranch.ratio");
   });
 
-  it("should display server-side errors but hide when field touched", async () => {
```

Comment:
We no longer need to test for this because errors clear `onChange`, test coverage is towards the top of the `useCommonForm` file.